CharacterCreation/03_Feats.txt

```diff
@@ -232,60 +232,60 @@ attacks. You also get a +10% on rolls to avoid blacking out.
 
 Point-man: 
 Prerequisites: PER 6
-Maybe you've been a point-man for a long time, maybe you don't mind the
+	Maybe you've been a point-man for a long time, maybe you don't mind the
 extra danger, or maybe you just room-clear a lot. Regardless, you are very
 good at what you do.
-When clearing a room, add 20 to your reflex roll.
+	When clearing a room, add 20 to your reflex roll.
 
 Preemptive Strike:  
 Prerequisites: DEX 7
-Whenever you are facing someone you are aware of, and they enter into adjacency
+	Whenever you are facing someone you are aware of, and they enter into adjacency
 with you, you gain a preemptive melee strike against them.
 
 Quickdraw: 
 Prerequisites: DEX 5
-As a free action, draw your chosen weapon(s) into firing position. If you are 
+	As a free action, draw your chosen weapon(s) into firing position. If you are 
 holding something, you can drop it and still draw your weapon as a free 
 action. However, you might not want to drop anything glass. Or any explosives.
 
 Rainman: 
 Prerequisites: CHA <=2 and INT >=8
-Your character has something going for them. Gain 1 non-Charisma skill @ 10 points
+	Your character has something going for them. Gain 1 non-Charisma skill @ 10 points
 (or add 10 points to a non-Charisma skill). This skill can exceed the skill point
 limit by 10. You may also ask the GM how many bullets each opponent has left in their
 weapon.
 
-Salty Hashtables
+Salty Hashtables:
 Prerequisites: Must be a mechanical race
-Your data inputs are sanitized and encrypted, complete with checksums!
+	Your data inputs are sanitized and encrypted, complete with checksums!
 Hacking you is 25% harder and you are aware whenever one of your senses is 
 malfunctioning.
 
-Salty Space Dog
+Salty Space Dog:
 Prerequisites: Pilot or 10 Piloting Skill
-You've been up in the black more times than you can count. You get a +15% to
+	You've been up in the black more times than you can count. You get a +15% to
 maintaining your bearings during a hyperspace jump as well as a +15% to stellar
 navigation.
 
 Shadow Sense:
 Prerequisites: Level 12 Monk with Shadow Walk or Shadow Strike.
-Your Aura Shield gains a +10 DR armour bonus if you have Shadow Strike or a +20 DR armour
+	Your Aura Shield gains a +10 DR armor bonus if you have Shadow Strike or a +20 DR armor

```

Comment:
Good catch! My eye would have missed this.

Reply:
Thanks! I thought we had already caught these, but I guess not! lol